accounts/tests.py

```diff
@@ -668,6 +668,17 @@ def test_user_delete_remove_sounds(self, delete_sound_solr):
         calls = [mock.call(i) for i in user_sound_ids]
         delete_sound_solr.assert_has_calls(calls, any_order=True)
 
+    def test_user_delete_using_form(self):
+        # This should set user's attribute active to false and anonymize it
+        user = self.create_user_and_content(is_index_dirty=False)
+        a = self.client.login(username=user.username, password='testpass')
+        form = DeleteUserForm(user_id=user.id)
+        encr_link = form.initial['encrypted_link']
+        resp = self.client.post(reverse('accounts-delete'),
+                {'encrypted_link': encr_link, 'password': 'testpass', 'delete_sounds': 'delete_sounds'})
+
+        self.assertEqual(User.objects.get(id=user.id).profile.is_deleted_user, True)
+
```

Comment:
Should probably add a failing test too, especially one where the password is mistyped...

Reply:
Yes, this test should be added.